projects/plugins/jetpack/class.jetpack.php

```diff
@@ -6302,6 +6307,35 @@ public function add_5_star_review_link( $plugin_meta, $plugin_file ) {
 		return $plugin_meta;
 	}
 
+	/**
+	 * Lazy instantiation of the Plugin_Tracking object.
+	 *
+	 * @return void
+	 */
```

Comment:
I wonder if it's worth adding the `@since $$next-version$$` to the new functions as well.

```suggestion
	/**
	 * Lazy instantiation of the Plugin_Tracking object.
	 *
	 * @since $$next-version$$
	 *
	 * @return void
	 */
```

Reply:
Good point, added the tags 👍